### PR TITLE
ConsoleInteraction : Correcting the typos

### PR DESCRIPTION
**Pull request by raul1991**
Corrected few of the typos in the file and added a document string
to a method.

closes issue https://github.com/coala/coala/issues/2791

### CODE REVIEW

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -316,7 +316,7 @@ def print_affected_files(console_printer,
                          file_dict,
                          color=True):
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: sooo for the commit message:

Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: Shortlog: tense is not correct, should be Imperative, aka: Correct typos
also, shouldn't be a space before the colon.

Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: Body: incorrect tense: should be Corrects, present tense. and it should be "a few", not "few".

also, the docstring add does not fit into this commit, so both remove it from the commit and from the commit body.

Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: It's simply ``Closes``, not ``closes issue``.

File: coalib/output/ConsoleInteraction.py [hunk 2]
```diff
@@ -423,6 +423,9 @@ def print_results(log_printer,
 
 
 def print_affected_lines(console_printer, file_dict, section, sourcerange):
+    """
+    Prints the lines affected by the bears.
+    """
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: need a separate commit :)

File: coalib/output/ConsoleInteraction.py (hunk 1 above)
Comment (raul1991) [inline coalib/output/ConsoleInteraction.py]: Okay

Comment (raul1991) [inline coalib/output/ConsoleInteraction.py]: Okay.

Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: The new shortlog is not what i said, it should be in present continuous..

Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: Closes, not closes.

**Comment by Adrianzatreanu** [comment]
ack c6d382a2984d092cf43c8a91e5c89b458cce5388

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**